### PR TITLE
Update working-with-the-rubygems-registry.md

### PR DESCRIPTION
**Pull request by mzagaja**
### Why:

The existing documentation is unnecessarily verbose and confusing.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Rather than having users edit their .gemrc it is likely easier to have them enter this simple command that does the same thing. I also cleaned up some formatting and added headers for improved readability.

I removed the reference to keycutter since it is otherwise not utilized in this documentation.

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md [hunk 1]
```diff
@@ -36,12 +36,6 @@ shortTitle: RubyGems registry
   Bundler version 1.13.7
   ```
 
-- Install keycutter to manage multiple credentials. To install keycutter:
```
Comment (thecontemoraryflorals) [inline content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md:39]: mzagaja:patch-1

File: content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md [hunk 2]
```diff
@@ -52,7 +46,12 @@ shortTitle: RubyGems registry
 
 {% data reusables.package_registry.required-scopes %}
 
-You can authenticate to {% data variables.product.prodname_registry %} with RubyGems by editing the  *~/.gem/credentials* file for publishing gems, editing the *~/.gemrc* file for installing a single gem, or using Bundler for tracking and installing one or more gems.
+You can authenticate to {% data variables.product.prodname_registry %} with RubyGems by: 
+* editing the  *~/.gem/credentials* file for publishing gems
+* adding a source using `gem sources --add` for installing gems using `gem install`
+* using Bundler for installing gems from an application Gemfile
+
+#### Publishing Gems
```
Comment (jhosman) [inline content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md:54]:
```suggestion
To publish and install gems, you can configure RubyGems or Bundler to authenticate to {% data variables.product.prodname_registry %} using your personal access token.
```

For further clarity, let's make this text more of an introductory summary, and focus on the details later in the section.

File: content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md [hunk 3]
```diff
@@ -63,7 +62,8 @@ For example, you would create or edit a *~/.gem/credentials* to include the foll
 :github: Bearer TOKEN
 ```
 
-To install gems, you need to authenticate to {% data variables.product.prodname_registry %} by editing the *~/.gemrc* file for your project to include `https://USERNAME:TOKEN@{% ifversion fpt or ghec %}rubygems.pkg.github.com{% else %}REGISTRY-URL{% endif %}/OWNER/`. You must replace:
+#### Installing Gems
```
Comment (jhosman) [inline content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md:65]:
```suggestion
```

Removed the headers for now to be consistent with our current style. I do understand what you were trying to do there, though. I'll open a followup internal issue for us to take a look at the current iteration of the content and make sure the structure isn't impeding comprehension. Thanks!

File: content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md [hunk 4]
```diff
@@ -72,18 +72,9 @@ To install gems, you need to authenticate to {% data variables.product.prodname_
   - `REGISTRY-URL` with the URL for your instance's Rubygems registry, `rubygems.HOSTNAME`. Replace *HOSTNAME* with the hostname of {% data variables.location.product_location %}.
 {% endif %}
 
-If you don't have a *~/.gemrc* file, create a new *~/.gemrc* file using this example.
-
+If you would like your package to be available systemwide you can simply run the following command to add your registry as a source.
```
Comment (jhosman) [inline content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md:75]:
```suggestion
If you would like your package to be available globally, you can run the following command to add your registry as a source.
```

**Review by jhosman — COMMENTED** [review]
Thanks for submitting this PR @mzagaja! Made a few minor edits, mostly for style consistency. I'll go ahead and apply those for you, and we'll get this merged soon! 🎉

**Review by SiaraMist — DISMISSED** [review]

File: content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md [hunk 5]
```diff
@@ -52,7 +46,7 @@ shortTitle: RubyGems registry
 
 {% data reusables.package_registry.required-scopes %}
 
-You can authenticate to {% data variables.product.prodname_registry %} with RubyGems by editing the  *~/.gem/credentials* file for publishing gems, editing the *~/.gemrc* file for installing a single gem, or using Bundler for tracking and installing one or more gems.
+To publish and install gems, you can configure RubyGems or Bundler to authenticate to {% data variables.product.prodname_registry %} using your personal access token.
```
Comment (jhosman) [inline content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md:49]:
```suggestion
To publish and install gems, you can configure RubyGems or Bundler to authenticate to {% data variables.product.prodname_registry %} using your {% data variables.product.pat_generic %}.
```

**Review by jhosman — APPROVED** [review]

**Pull request merged**